__tests__/app.test.js

```diff
@@ -42,3 +42,44 @@ describe("GET /api/topics", () => {
       });
   });
 });
+
+describe("GET 200 /api/articles/:article_id", () => {
+  test("get 200", () => {
+    return request(app)
+      .get("/api/articles/2")
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article).toBeInstanceOf(Object);
+        expect(Object.keys(article).length).toBe(8);
+        expect(article).toEqual({
+          author: expect.any(String),
+          title: expect.any(String),
+          article_id: expect.any(Number),
+          body: expect.any(String),
+          topic: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String),
+        });
+      });
+  });
+  test("STATUS 400 responds with a 400 and correct message for an invalid article", () => {
+    return request(app)
+      .get("/api/articles/hello")
+      .expect(400)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toBe("Sorry, this is not a valid article id");
```

Comment:
I love the politeness of these messages, but don't worry about making them user-focused. The client will often be another dev, so something functional also works (e.g. invalid article id)